web/html/src/components/input/Select.js

```diff
@@ -1,12 +1,23 @@
 // @flow
 
 import * as React from 'react';
+import ReactSelect from 'react-select';
 import { InputBase } from './InputBase';
 import { FormContext } from './Form';
 
 type Props = {
-  /** <option> elements to display */
-  children: React.Node,
+  /** Select options */
+  options: Array<Object>,
```

Comment:
IIUC, it's allowed to have strings in options. Maybe define it as something like:
```
type OptionType = {
  value: number | string,
  label: string
}
```
and then in `type Props`:
```
options: Array<OptionType | string>
```

Reply:
well, `OptionType` wouldn't fly since you can have any object as long as you set the `getOptionValue` and `getOptionLabel` properties. but `options: Array<Object | string>` yes